README.md

```diff
@@ -1,71 +1,107 @@
-# rex
+# REX
 
-rex is a Really EXcellent way to develop webapps.
+REX is a Really EXcellent way to develop web-apps.
 
-# Quickstart
+## Quick start
 
-First, (install Docker)[https://docs.docker.com/install/].
+First, [install Docker](https://docs.docker.com/install/).
 
 Next, run
 
-```
-npm run db:sync-force
-npm start
+```bash
+$ npm run docker:db:sync:force
+$ npm start
 ```
 
-To get a development environment. The first time you run these
-commands we'll build a bunch of docker images and do a bunch of
-initialization - please be patient.
+To get a development environment. The first time you run these commands we'll build a bunch of docker images and do a bunch of initialization - please be patient.
```

Comment:
I prefer the line breaks as written - it helps keep this text readable pre-rendering (I usually consume these documents as written rather than as rendered to HTML, so this one is dear to my heart ;-p )